language-pages/color.md

```diff
@@ -11,4 +11,8 @@ sidenav: language-side-nav
 - Understanding color grades and accessibility
 - Defining theme and state tokens (unless they don't work for your project)
 - Unless you have to use specific brand color values, consider using colors from the system token palette to benefit from the accessible contrast features.
-  - Use the [USWDS Color Tool](https://civicactions.github.io/uswds-color-tool/) from Civic Actions to find system token colors similar to existing colors on a project that you may want to migrate.
\ No newline at end of file
+  - Use the [USWDS Color Tool](https://civicactions.github.io/uswds-color-tool/) from Civic Actions to find system token colors similar to existing colors on a project that you may want to migrate.
+
+
+![import button on tokens plugin]({{ site.baseurl }}/assets/img/color-infographic.png)
```

Comment:
This is helpful, but `button-primary-background` is not a USWDS token, at least that I'm aware of. Some options that are relevant here are "variants" (like those available on [buttons](https://designsystem.digital.gov/components/button/#component-variants-button)) or utility classes, which is what you would use to customize a component (for example, `bg-primary-dark` or `bg-accent-cool`).

Reply:
Needs alt text that describes the content (and/or context) of the image